cozumler/en-yakin-noktalar/cozum.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import math
+from math import sqrt
+
+coordinate = [(3, 1), (3, 4), (4, 2)]
+
+def distances(coordinate):
+    values = []
+    parsing = []
+    distance = []
+    for i in range(len(coordinate)):
+        parsing.append(" ".join(str(x) for x in coordinate[i]))
+        calculateValue = parsing[i].split(" ")
+        firstValue = math.pow(int(calculateValue[0]), 2)
+        secondValue = math.pow(int(calculateValue[1]), 2)
+        distance.append(sqrt(int(firstValue) + int(secondValue)))
+    return distance
+
+def calculate():
+    data = distances(coordinate)
+    return data
+
+
+def matchResult(coordinate):
```

Comment:
PEP8'e uyarsak güzel olabilir function ve variable isimlerinde

(Function names should be lowercase, with words separated by underscores as necessary to improve readability.)